lib/rules/no-eol-whitespace/index.js

```diff
@@ -204,9 +204,11 @@ const rule = (primary, secondaryOptions, context) => {
 				}
 
 				if (isDeclaration(node)) {
-					if (node.raws.value) {
-						fixText(node.raws.value.raw, (fixed) => {
-							node.raws.value.raw = fixed;
+					const rawsValue = node.raws.value;
```

Comment:
[note] `node.raws.value` is now an optional property.

Reply:
Nice. I saw you raise the issue upstream. Thanks for that!